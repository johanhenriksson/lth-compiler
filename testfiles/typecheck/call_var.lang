int actual_func() { }
int main() {
    int variable;
    variable(5); // error
    actual_func();
}
