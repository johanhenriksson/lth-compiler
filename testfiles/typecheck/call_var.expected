Error at line 4,5: cannot call 'variable': not a function
